### ISSUE TITLE
Overflow Menu Accessibility support 

### ISSUE THREAD

**Issue by aradhikaIbm** [IBM/carbon-components-angular#62]
## Detailed description

> Describe in detail the issue you're having.
Looking at using the overflow menu as we are having accessibility issues. Are there plans to add the keyboard accessible function to the menu? 

Per the guideline: (https://www.w3.org/TR/wai-aria-practices/#menu)
We need support for the arrows, end and home. 

Also I noticed while using the Storybook i cannot tab to get focus on the menu to able to use the menu with keyboard only. 

> Is this a feature request (new component, new icon), a bug, or a general issue?
General accessibility issue with overflow menus.

> Is this issue related to a specific component?
Overflow menu

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IBM Watson Recruitment, Accessibility deadline of Dec 10, 2018.

**Comment by aradhikaIbm** [comment]
Looks like the menu does not focus on the first item as well

**Comment by alwilkin** [comment]
Can we get an ETA on fix?

**Comment by youda97** [comment]
## DAP Analysis

Level | Description
-- | --
V | Elements that use 'role' must reference a valid WAI-ARIA role. | Details
V | The WAI-ARIA role(s) and/or attribute(s) aria-expanded are not valid for the element div. See rule help for guidance. | Details
V | All content must reside within a WAI-ARIA landmark or labelled region role. | Details
V | Widget with role of button must have at least one tabbable element.

**Comment by tw15egan** [comment]
FYI, I just merged some a11y improvements to the vanilla Overflow Menu that may help in resolving these DAP issues: https://github.com/IBM/carbon-components/commit/c7bdb3105e128d6e7f3253d25769979e6aa15c37

**Comment by youda97** [comment]
Thank you @tw15egan !!
